packages/lesswrong/server/repos/UsersRepo.ts

```diff
@@ -543,6 +543,50 @@ class UsersRepo extends AbstractRepo<"Users"> {
     `, [limit])
   }
 
+   async getUsersForInactiveSummaryEmail(limit: number): Promise<(DbUser & {
+     fetchActivitySince: Date,
+   })[]> {
+    return this.getRawDb().manyOrNone(`
+      -- UsersRepo.getUsersForInactiveSummaryEmail
+      SELECT
+        u.*,
+        GREATEST(
+          u."inactiveSummaryEmailSentAt",
+          rs."max_last_updated"
+        ) "fetchActivitySince"
+      FROM public."Users" AS u
+      LEFT JOIN (
+        SELECT "userId", MAX("lastUpdated") AS max_last_updated
+        FROM "ReadStatuses"
+        WHERE "isRead" IS TRUE
```

Comment:
```suggestion
```
I would cut this just to simplify the query, marking a post as unread can still count as activity